packages/material-ui-lab/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://material-ui.com/components/about-the-lab",
   "scripts": {
-    "build": "yarn build:cjs && yarn build:esm && yarn build:es && yarn build:types && yarn build:copy-files",
+    "build": "yarn build:cjs && yarn build:esm && yarn build:es && yarn build:copy-files && yarn build:types",
```

Comment:
ASAIR this was required because `copy-files` fails if did not found the ts declaration

Reply:
There was also a change to `scripts/copy-files` which I reverted. `yarn workspace @material-ui/lab build` is now passing. So if you changed something then not just because the script was failing.